src/Tribe/Ticket_Object.php

```diff
@@ -1080,6 +1080,28 @@ public function get_event_id() {
 			}
 			return $this->event_id;
 		}
+
+		/**
+		 * Check if the ticket has meta enabled.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool
+		 */
+		public function has_meta_enabled() {
+			$has_meta = get_post_meta( $this->ID, '_tribe_tickets_meta_enabled', true );
+
+			/**
+			 * Filters if the ticket has meta or not.
+			 *
+			 * @param bool $has_meta Wether if it has meta.
+			 * @param int  $ticket_id The ticket ID.
+			 */
+			$has_meta = apply_filters( 'tribe_tickets_has_meta_enabled', $has_meta, $this->ID );
+
+			return tribe_is_truthy( $has_meta );
```

Comment:
```suggestion
			return (boolean) apply_filters( 'tribe_tickets_has_meta_enabled', $has_meta, $this->ID );
```

Reply:
I felt that by using `tribe_is_truthy` we could contemplate more "bool" options, like `yes`||`no` as strings. But I'm ok with the changes.

Reply:
I made this suggestion because that's our chosen DB storage workaround. It's a filter result and we should definitely stick to expecting and enforcing a normal boolean especially with the filter itself saying the param is a boolean.

Reply:
👍Perfect! Makes sense. I thought of the case that they could use a string, that was the reason of using the `truthy` and not casting.